app/models/miq_worker.rb

```diff
@@ -316,19 +316,41 @@ def queue_name
     end
   end
 
-  def self.build_command_line(guid)
-    command_line = "#{Gem.ruby} #{runner_script} --heartbeat --guid=#{guid} #{name}"
-    ENV['APPLIANCE'] ? "nice #{nice_increment} #{command_line}" : command_line
+  def self.build_command_line(options)
```

Comment:
We previously passed a guid, now an options hash so this could break anyone using this seemingly private interface.

To keep this PR small, I was going to mark this method as private and update the tests of it as a followup.